src/setup_runs/config_read_functions.py

```diff
@@ -37,58 +37,34 @@ def boolean_converter(
 
     return value.lower() in truevals
 
-
-def read_config_file(configFile: str) -> str:
-    """
-    Read and return the content of a configuration file.
-
-    Parameters
-    ----------
-    configFile
-        The path to the configuration file.
-
-    Returns
-    -------
-        The content of the configuration file.
-
-    """
-
-    assert os.path.exists(
-        configFile
-    ), f"No configuration file was found at {configFile}"
-
-    try:
-        with open(configFile, "rt") as f:
-            input_str = f.read()
-        return input_str
-    except Exception as e:
-        print("Problem reading in configuration file")
-        print(e)
-        sys.exit()
-
-
-def parse_config(input_str: str) -> dict[str, str | bool | int]:
-    """
-    Parse the input string  and remove comments.
-
-    Parameters
-    ----------
-    input_str
-        The input string containing configuration data.
-
-    Returns
-    -------
-        The parsed configuration data.
-    """
-
-    try:
-        # strip out the comments
-        input_str = re.sub(r"#.*\n", "\n", input_str)
-        return json.loads(input_str)
-    except Exception as e:
-        print("Problem parsing in configuration file")
-        print(e)
-        sys.exit()
+#
+# def read_config_file(configFile: str) -> str:
```

Comment:
Is this still needed? Otherwise it can be deleted

Reply:
It's deleted.